src/classes/presentation_controller.vala

```diff
@@ -2174,6 +2174,12 @@ namespace pdfpc {
                 this.presenter.show_qrcode_window(false);
             }
         }
+#else
+        public void show_qrcode() {
+	  GLib.printerr("No REST support, QR disabled.\n");
```

Comment:
We use 4-char software tabs for indentation.

Reply:
> We use 4-char software tabs for indentation.

Oups, my bad :sweat_smile:.  I changed it in the newest commit.